irc/server.go

```diff
@@ -407,7 +407,7 @@ func (server *Server) playRegistrationBurst(session *Session) {
 
 	modestring := c.ModeString()
 	if modestring != "+" {
-		session.Send(nil, d.nickMask, RPL_UMODEIS, d.nick, modestring)
+		session.Send(nil, c.server.name, RPL_UMODEIS, d.nick, modestring)
```

Comment:
hmm, we should be able to use `server.name` here like we do above, but otherwise looks good!